CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/gui/ValueInputPane.java

```diff
@@ -160,11 +160,8 @@ public void updateItem(final String item, final boolean empty) {
             field = new SpellCheckingTextArea(parameter.isSpellCheckEnabled());
             if (suggestedHeight > 1) {
                 field.setWrapText(true);
-            } else {
```

Comment:
Should probably add this back?

Reply:
Verified after removing this block and works as expected